PlayCover/View/AppSettingsView.swift

```diff
@@ -29,66 +29,78 @@ struct AppSettingsView: View {
         VStack(alignment: .leading) {
             VStack(alignment: .leading) {
                 HStack {
-                    Toggle(NSLocalizedString("settings.toggle.km", comment: ""), isOn: $keymapping).padding()
-                    Toggle(NSLocalizedString("settings.toggle.gaming", comment: ""), isOn: $gamingMode).padding()
+                    Toggle(
+                        NSLocalizedString(
+                            "settings.toggle.km", comment: ""
+                        ), isOn: $keymapping
+                    )
+                    .padding(5)
+                    .padding(.leading, 10)
+                    Toggle(
+                        NSLocalizedString(
+                            "settings.toggle.gaming", comment: ""
+                        ), isOn: $gamingMode
+                    )
+                    .padding(5)
+                    .padding(.leading, 10)
```

Comment:
If you're adding this same padding to everything, you should do it on the stack itself to be concise.